scripts/qa_crowbarsetup.sh

```diff
@@ -1390,15 +1390,9 @@ EOF
     zypper rl kernel-default
 
     if [ -z "$NOINSTALLCLOUDPATTERN" ] ; then
-        zypper --no-gpg-checks -n in -l -t pattern cloud_admin
-        local ret=$?
-
-        if [ $ret = 0 ] ; then
-            echo "The cloud admin successfully installed."
-            echo ".... continuing"
-        else
-            complain 86 "zypper returned with exit code $? when installing cloud admin"
-        fi
+        safely zypper --no-gpg-checks -n in -l -t pattern cloud_admin
+        # make sure to use packages from PTF repo (needs zypper dup)
+        safely zypper -n dup -r cloud-ptf
```

Comment:
`zypper dup --help` is your friend.

"dup --from" is most likely the better solution for this PR (but it worked anyhow in our situation).
Are you proposing "--from" it or are you just questioning?